invenio_accounts/ext.py

```diff
@@ -238,6 +238,12 @@ def init_config(self, app):
             )
         )
 
+        # Follow SETTINGS_TEMPLATE pattern
```

Comment:
Could you set a breakpoint and check if `SETTINGS_TEMPLATE` is already initialised? Also could you do this for blueprint? We tried it locally and it seems to be set already in the blueprint.